src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -290,7 +265,7 @@ protected override void OnScriptFileChanged(object sender, FileSystemEventArgs e
                     try
                     {
                         fileTraceWriter.Info(text);
-                        traceWriter.Info(text);
+                        invocationContext.TraceWriter.Info(text);
```

Comment:
If the `TraceWriter` is optional (as stated here https://github.com/Azure/azure-webjobs-sdk-script/pull/604/files#diff-5068ee687748ecb1d19219ddd4287b90R97), we need to be defensive here.
Should the `TraceWriter` be optional, though?